.github/workflows/main.yml

```diff
@@ -11,7 +11,7 @@ env:
   TEST_TARGET: testing
 
   # DO_TEST - true to build and run unit tests, false to skip the tests
-  DO_TEST: false
+  DO_TEST: true
```

Comment:
Thank you!

Reply:
strange that this is even an option

Reply:
I implemented that when I first started working on github actions.  It was a more generic action and not everything had tests that existed or succeeded at that point in time.  It can be removed if no longer necessary.